demo/src/screens/componentScreens/PickerScreen.tsx

```diff
@@ -137,7 +137,8 @@ export default class PickerScreen extends Component {
           <Picker
             title="Native Picker"
```

Comment:
Change the title too. Also, don't you want to keep both examples for native and wheel pickers?

Reply:
Well, I want to deprecate useNativePicker, you mean for the sake of sanity checks - to see nothing was broken with useNativePicker prop?

Reply:
Sure, if you're deprecating the 'useNativePicker' now there's no point having its example (but the title should be changed to avoid confusion).